src/components/InventoryGroups/Modals/AddSystemsToGroupModal.js

```diff
@@ -71,6 +81,16 @@ const AddSystemsToGroupModal = ({
     [isModalOpen]
   );
 
+  const calculateSelected = () => (selected ? selected.size : 0);
+
+  const handleModalClose = () => {
+    setIsModalOpen(false);
+    if (calculateSelected() > 0) {
+      dispatch(selectEntity(-1, false));
+    }
+    dispatch(setFilter([]));
```

Comment:
nit-pick: I think you can use CLEAR_FILTERS action or something similar, it will erase the applied filters and also selections for InventoryTable.

Reply:
I've found `CLEAR_ENTITIES` but it doesn't set selected to empty, but replaced the `setFilter([])` with `clearFilter()`